src/SymfonyCache/TagSubscriber.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace FOS\HttpCache\SymfonyCache;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use FOS\HttpCache\SymfonyCache\Tag\ManagerInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\HeaderBag;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Adds tag invalidation capabilities to the Symfony HTTP cache
+ *
+ * @author Daniel Leech <daniel@dantleech.com>
+ * 
+ * {@inheritdoc}
+ */
+class TagSubscriber implements EventSubscriberInterface
+{
+    /**
+     * HTTP method to use for remote tag invalidation.
+     */
+    const METHOD_INVALIDATE = 'INVALIDATE';
+    
+    /**
+     * Name for HTTP header containing the tags (for both invalidation and
+     * initial tagging).
+     */
+    const HEADER_TAGS = 'X-TaggedCache-Tags';
+
+    /**
+     * Header which should contain the content digest produced by the Symfony
+     * HTTP cache.
+     */
+    const HEADER_CONTENT_DIGEST = 'X-Content-Digest';
```

Comment:
I initially coded these as options, as with the `UserContextSubscriber`, but it makes it awkward when you need these values from somewhere else, and would indicate the need for a `Configuration` object or something..